### ISSUE TITLE
Indentation of lambda body inside nested namesapce

### ISSUE THREAD

**Issue by Yadunund** [uncrustify/uncrustify#2678]
[//]: # "  If the issue is connected to existing Uncrustify options please, if possible, add the"
[//]: # "  following information to ease up the process:"
[//]: # "  • a link to a debug file:"
[//]: # "      generated with: 'uncrustify -p debug.txt -c pathToUsedConfig.cfg -f toBeFormatedFile.cpp' "
[//]: # "      Example: [debug.txt](https://linkToTheFile)"
[//]: # "      Example hosters for debug files: pastebin.com, gist.github.com, ..."
[//]: # "      The used config file is included in the debug file and does not need to be included here."
[//]: # 
[//]: # "  • include a small but complete test file that will be uncrustifyed"
[//]: # "  • include the generated results"
[//]: # "  • include the expected results"
[//]: # 
[//]: # "  ✋ please add a line containing ``` above and below of each of those three code sections"
[//]: # 
[//]: # "  • include the current version of your Uncrustify executable"
[//]: # "      printout via 'uncrustify -v'"
[//]: # "      Example: current version: uncrustify 0.63"
[//]: # "      or if possible additionally with the git sha of the commit"
[//]: # "                    current version: uncrustify 0.63 dc7b412"
[//]: # 
[//]: # "  • if possible include a version that worked"
[//]: # "      Example: working version: uncrustify 0.63"
[//]: # "      or"
[//]: # "                    working version: uncrustify 0.63 2a5e88f"

Hi,

When my lambda is defined inside a single namespace, the config file formats the output as expected. However, when inside a nested namespace, the lambda braces indent to a position below the lambda capture block.

Uncrustify version : `Uncrustify-0.69.0`

With the attached
[config.zip](https://github.com/uncrustify/uncrustify/files/4260597/config.zip)

 configuration file, when the input is 

```
namespace p {
void some_func()
{
  auto a = b(
    c,
    []() -> bool
  {
      dosmth();
  });
}
} // namespace p
```

`The output is without change which is desired.`

However, when the same function is inside a nested namespace, ie `Input:`
```
namespace p {
namespace q {
void some_func()
{
  auto a = b(
    c,
    []() -> bool
  {
      dosmth();
  });
}
} // namespace q
} // namespace p
```

`New Output:`
```
namespace p {
namespace q {
void some_func()
{
  auto a = b(
    c,
    []() -> bool
    {
      dosmth();
    });
}
} // namespace q
} // namespace p
```
The braces after the lambda signature shift to below the capture block.

**Comment by gmaurel** [comment]
The problem is already solved at the current version.

**Comment by Yadunund** [comment]
Thank you very much for pointing this out. I can confirm that the problem is solved as per version `Uncrustify_d-0.70.1`
